src/term.rs

```diff
@@ -1497,6 +1497,14 @@ pub enum NAryOp {
     ///   - a [record](Term::Record), which is the record we wish to seal the tail into,
     ///   - the [term](Term) that we wish to seal.
     SealRecordTail(),
+    /// Unseals a term from the tail of a record and returns it.
+    ///
+    /// Takes three arguments:
+    ///   - the [sealing key](Term::SealingKey), which was used to seal the tail,
+    ///   - a [label](Term::Label) which will be used to assign blame correctly if
+    ///     something goes wrong while unsealing,
+    ///   - the [record](Term::Record) whose tail we wish to unseal.
+    UnsealRecordTail(),
```

Comment:
The same consideration as for `%record_empty_with_tail%`, maybe it would be better to go with `RecordUnsealTail`?